_test/closure10.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+func main() {
+	foos := []func(){}
+
+	for i := 0; i < 3; i++ {
+		a, b := i, i
+		_ = b
+		foos = append(foos, func() { println(i, a) })
```

Comment:
You could do:
```suggestion
		foos = append(foos, func() { println(i, a, b) })
```
to prove that b actually has the right value